ideascube/widgets.py

```diff
@@ -1,6 +1,101 @@
 from django.forms import widgets
 
 from ideascube.configuration import get_config
+from django.utils.html import mark_safe, escape
+from django.utils.encoding import force_text
+from django.forms.utils import flatatt
+
+import bleach
+
+
+_script_template = '''<script type="text/javascript">tinymce.init({{
+selector: "#{widget_id}",
+inline: true,
+theme: "inlite",
+hidden_input: false,
+menubar: false,
+toolbar: false,
+selection_toolbar: {settings[selection_toolbar]},
+insert_toolbar: {settings[insert_toolbar]},
+plugins: {settings[plugins]}{settings[extra_settings]}
+}});</script>'''
+
+
+class RichTextEntry(widgets.Widget):
+    def __init__(self, *,
+            with_media = False,
+            selection_toolbar=None,
```

Comment:
That's a good question :)

Mainly because, in my first version, users could (and should) define `insert_toolbar` and `plugins`.
But now that I calculate them from the `with_media` option, I remove them but I keep the `selection_toolbar` customizable. 
We never use this option when we use the widget, so maybe we could simply remove it and force the `selection_toolbar` ourselves.